storage/ndb/mcc/frontend/dojo/dijit/package.json

```diff
@@ -6,11 +6,7 @@
 	},
 	"main": "main",
 	"dependencies": {
-<<<<<<< HEAD
 		"dojo":"1.16.2"
-=======
-		"dojo": "1.15.4"
->>>>>>> mysql-server/8.0
 	},
```

Comment:
since we don't care about ndb, shouldn't we take upstream version changes. ie. 1.15.4 ? wait did they lower version requirements?
HEAD is our source and mysql-sever/8.0 is upstream 8.0 right? :thinking:

Reply:
I got the recommendation from GitHub to update the version of the dojo as a security alert. It was done in the previous release. so should do the changes according to upstream or let it be 1.16.2 to avoid security alert of the repo

Reply:
I think we leave it same as upstream. check with kamil.

Reply:
leaving it as upstream. will check with team if GitHub complains again.